pallets/did/src/lib.rs

```diff
@@ -100,9 +101,19 @@ decl_module! {
 	}
 }
 
+#[derive(Encode, Decode)]
+pub struct Did<T: Config> {
```

Comment:
Ah now I see. The stuff i suggest below won't work. This thing is not the DID but something different. I think it's the did document https://www.w3.org/TR/did-core/#dfn-did-documents

```suggestion
pub struct DidDocument<T: Config> {
```

Reply:
Oh thanks, I was unhappy with the naming. Fixed in 79bd337

Reply:
Ähm I just talked to raphael and it's not the document but more like the `DidRecord`. Sorry 😭 !!

Reply:
There is no 'DidRecord' AFAIK but that name is better suited...